docs/users/installation.md

```diff
@@ -18,13 +18,23 @@ supported. Make sure you follow the "sbt 1.3x and newer" instructions below.
 
 ## sbt
 
-Start by installing the sbt plugin in `project/plugins.sbt`
+Start by installing the sbt 1.x plugin in `project/plugins.sbt`
 
 ```scala
 // project/plugins.sbt
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "@VERSION@")
 ```
 
+> sbt-scalafix is no longer published for sbt 0.13.x. You should be able to run
+> the latest version of Scalafix with the final sbt-scalafix version published
+> for sbt 0.13.x, but all features documented below might not be supported.
+>
+> ```scala
+> // project/plugins.sbt
+> addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.29") // final sbt 0.13.x version
+> dependencyOverrides += "ch.epfl.scala" % "scalafix-interfaces" % "@VERSION@"
```

Comment:
let's hold this PR until the next release, as it will look weird to have the same version on both lines right now

Reply:
I agree